ocpp/v16/call.py

```diff
@@ -229,3 +234,72 @@ class DataTransferPayload:
     vendor_id: str
     message_id: Optional[str] = None
     data: Optional[str] = None
+
+
+# Backported CALL Messages
+# Improved security for OCPP 1.6-J specification
+
+@dataclass
+class CertificateSignedPayload:
+    cert: List
+
+
+@dataclass
+class DeleteCertificatePayload:
+    certificate_hash_data: Dict
+
+
+@dataclass
+class ExtendedTriggerMessagePayload:
+    requested_message: MessageTriggerType
+
+
+@dataclass
+class GetInstalledCertificateIdsPayload:
+    type_of_certificate: CertificateUseType
+
+
+@dataclass
+class GetLogPayload:
+    log: Dict
+    log_type: LogType
+    request_id: int
+    retries: Optional[int] = None
+    retry_interval: Optional[int] = None
+
+
+@dataclass
+class InstallCertificatePayload:
+    certificate_type: CertificateUseType
+    certificate: str
+
+
+@dataclass
+class LogStatusNotificationPayload:
+    status: LogStatusType
+    request_id: int
+
+
+@dataclass
+class SecurityEventNotificationPayload:
+    type: str
+    timestamp: str
+
```

Comment:
Please add the missing field `tech_info`.
![Uploading Screenshot_2021-04-26_11-22-18.png…]()